compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -1030,6 +1031,33 @@ public void visit(BLangConstrainedType constrainedTypeNode) {
         resultType = constrainedType;
     }
 
+    private BType getXMLConstrainedType(BType constraintType, DiagnosticPos pos) {
+        BType constrainedType;
```

Comment:
Shall we remove this and return the result then and there.
Difficult to differentiate `constrainedType` and `constraintType`.

Reply:
Removed the method.